commcare_connect/opportunity/tasks.py

```diff
@@ -360,3 +360,19 @@ def bulk_update_payments_task(self, opportunity_id: int, headers: list[str], row
         messages = [f"Payment Import failed: {e}"] + getattr(e, "invalid_rows", [])
 
     set_task_progress(self, f"<br>".join(messages), is_complete=True)
+
+
+@celery_app.task(bind=True)
+def bulk_update_visit_status_task(self, opportunity_id: int, headers: list[str], rows: list[list]):
```

Comment:
same comment on the other one about locking. I think its important only one of these is running per opportunity to avoid race conditions in the accrual calculations